doc/source/whatsnew/v0.19.0.txt

```diff
@@ -332,6 +332,7 @@ API changes
 - ``PeriodIndex.fillna`` with ``Period`` has different freq now coerces to ``object`` dtype (:issue:`13664`)
 - More informative exceptions are passed through the csv parser. The exception type would now be the original exception type instead of ``CParserError``. (:issue:`13652`)
 - ``astype()`` will now accept a dict of column name to data types mapping as the ``dtype`` argument. (:issue:`12086`)
+- `Series.sort_index`` will now accept kwargs ``kind`` and ``na_position`` (:issue:`13589`)
```

Comment:
can you update the Series (only) doc-string to say kind/na_position were added in 0.19.0.